GTScript-Parallel-model.md

```diff
@@ -42,7 +42,8 @@ The effect of the program is as if statements are executed as follows:
 3. every vertical _interval_ is executed as a sequential for-loop over the `K`-range following the order defined by the iteration policy,
 4. within a computation, it is illegal to write to an external (non-temporary) field (or aliases pointing to the same memory location) if it is also read with horizontal offset; this rule does not apply to temporaries,
 5. in an assignment, a field cannot be assigned to, if it's used in the r.h.s. expression with a horizontal offset; note that this rule only applies to temporaries as this pattern is already excluded for external fields by rule 4,
-6. for `if`-`else` statements, the condition is evaluated first, then the `if` and `else` bodies are evaluated with the same rules as above.
+6. for `if`-`else` statements, the condition is evaluated first, then the `if` and `else` bodies are evaluated with the same rules as above,
+7. execution of a program is illegal if any access in any branch is outside of array bounds.
```

Comment:
The new point contains the only mention to `arrays` in the document, which might be a bit confusing since everywhere else `field` is used. My suggestion to improve it:
```suggestion
7. execution of a program is illegal if any field access in any branch is outside of array bounds.
```